### DISCUSSION TITLE
command line useage

### DISCUSSION THREAD

**Discussion by omega3** [snakers4/silero-models#267] [General]
I don't know how to use colab. I don't have IT background. 

Is it possible to use it on Linux command line?
I installed via pip:
```
pip install silero
import silero
```
and `silero --help` shows: `command not found`

I want to use text to speech. I don't know how to produce wav file on my PC, possibly using ssml tags for sentence breaks. I am interested in English voices. 
Do I need to run a python script for this? Can you share an example?

Do silero models can be used in other projects like piper, coqui-tts?

**Comment by zoldaten** [comment]
@omega3 it is not working in cmd "out of the box" like piper. but one can make one-file-exe to use it.
for examples see Colabs.
for you case:
```
import os
import sys
import torch
from playsound import playsound

##if len(sys.argv) != 3:
##    print ("Syntax: python3 text_to_wav.py text.txt voice.wav")
##    exit()

input_file_name = 'test.txt' #sys.argv[1]
output_file_name = 'voice.wav'#sys.argv[2]

print ("Input file:", input_file_name)
print ("Output file:", output_file_name)

with open(input_file_name, 'r', encoding='utf-8') as file:
    example_text = file.read().rstrip()

print("Text:")
print(example_text)

print("\r\nPocessing...")

device = torch.device('cpu')
torch.set_num_threads(4)

local_file = 'v4_en.pt'
#remote_file = 'https://models.silero.ai/models/tts/ua/' + local_file
sample_rate = 48000
speaker='mykyta' #!change speaker

#if not os.path.isfile(local_file):
    #torch.hub.download_url_to_file(remote_file, local_file)

model = torch.package.PackageImporter(local_file).load_pickle("tts_models", "model")
model.to(device)

audio_paths = model.save_wav(text=example_text, speaker=speaker, sample_rate=sample_rate, audio_path = output_file_name)

#print("Playing...")

#playsound(output_file_name)

#print("Done.")

```
change speaker there.

> Do silero models can be used in other projects like piper, coqui-tts?

no need. it is standalone project.
